### DISCUSSION TITLE
Orchestrate Uno CI on Azure

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#6820] [Q&A]
Will you add Uno specific CI template to Azure DevOps pipeline configuration ?
Or shall I chose `.NET Desktop` or `Universal Windows` ?

![image](https://user-images.githubusercontent.com/43340959/130300100-450b9eac-207f-46ad-a5d4-a4e6b268e562.png)

**Comment by jeromelaban** [comment]
We do not have a template for this yet, but you can [use this repository](https://github.com/unoplatform/Uno.QuickStart/blob/master/.vsts-ci.yml) as an example for running in Azure Devops.
